### PR TITLE
add SentinelAddrs

### PR DESCRIPTION
**Pull request by yushizhao**
related issue https://github.com/mediocregopher/radix/issues/114

### CODE REVIEW

File: sentinel.go [hunk 1]
```diff
@@ -205,6 +205,20 @@ func (sc *Sentinel) Addrs() (string, []string) {
 	return sc.primAddr, secAddrs
 }
 
+// SentinelAddrs returns the currently known network address of sentinels
```
Comment (mediocregopher) [inline sentinel.go]: Wording it this way sounds a little better :)

```
SentinelAddrs returns the addresses of all known sentinels.
```

**Comment by yushizhao** [comment]
Okay! A new commit is made to reflect these comments. And I am more than happy to discuss the right way to determine whether a node is down. I am open the new issue now.

**Comment by mediocregopher** [comment]
Thanks! Do you mind fixing the wording on the method doc? Since it's publicly facing documentation I'm a little picky with it :)

File: sentinel.go (hunk 1 above)
Comment (yushizhao) [reply sentinel.go]: Sure

**Comment by yushizhao** [comment]
Here you go! By the way, I like the doc.

**Comment by mediocregopher** [comment]
Thanks!

**Pull request merged**